incubator/hnc/plugin/hnc.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: hnc
+spec:
+  version: "v0.0.1"
```

Comment:
We're calling the next release "v0.1" so let's keep them in sync. Alternatively, you can put some bogus word in here like VERSION and then use something like `sed` to replace it from the makefile (where the makefile itself gets the value from an env var, something like `VERSION=v0.1 make krew-build`).

Reply:
Any reason we aren't using semver like most other k8s versions.

Reply:
> Any reason we aren't using semver like most other k8s versions.

+1